db/schema.rb

```diff
@@ -88,28 +88,15 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "buildings", force: :cascade do |t|
```

Comment:
@nomadicoder What's this?

Reply:
Looks like something got out of sync between this working version and changes yet to be merged into master. I rebased with latest master and resolved the conflicts.